src/components/App/AppHeader/DefaultHeader/index.vue

```diff
@@ -2,7 +2,7 @@
   <div class="DefaultHeader">
     <template v-if="mobileMode">
       <div class="DefaultHeader-user-info">
-        <img class="DefaultHeader-user-info-avatar" :src="avatarUrl" />
+        <img class="DefaultHeader-user-info-avatar" :src="avatarUrl" alt="" />
```

Comment:
Are we able to put an alt-text in here or does `html-validate` show some errors if so? I'm thinking maybe something like "user avatar"?

Reply:
If we set `alt="user avatar"`, a screen reader will say "user avatar" in addition to the user's name which is already shown; this is likely to be distracting, as it doesn't convey any meaningful information to a blind or visually impaired person.

In cases like this where the image is decorative, i.e. the image is not meant to convey any information that would be meaningful to a blind or visually impaired person, the WCAG guidelines specify that `alt=""` should be set explicitly rather than omitted; this way screen readers will know to simply proceed to reading the user's name. If the `alt` attribute is missing, many screen readers will assume that we simply forgot to put it in there and read the image filename aloud.

Reply:
That was very informative, thank you!